.github/workflows/testAndPublish.yml

```diff
@@ -386,13 +402,67 @@ jobs:
           GH_TOKEN: ${{ github.token }}
           GH_REPO: ${{ github.repository }}
 
+  deploySnapshot:
```

Comment:
we may need to skip this job if merging before the deploy hook is ready. I suggest we enable snapshots first, and then releases

Reply:
actually - there is no danger in firing off this deployment event. As long as we aren't doing it at the same time as appveyor